tools/disciplina-tools.cabal

```diff
@@ -1,46 +0,0 @@
-name:                disciplina-tools
-version:             0.0.1
-synopsis:            Disciplina - Tools for research and development
-description:         This package contains tools for development of Disciplina
-                     platform, such as simulators of cryptographic protocols.
-author:              Serokell
-maintainer:          hi@serokell.io
-copyright:           2018 TeachMePlease.com
-cabal-version:       >=1.10
-
-executable disciplina-simulacrum
-  hs-source-dirs:      src/simulacrum
-  main-is:             Main.hs
-  build-depends:       universum
-                     , base >= 4.9
-                     , mtl
-
-  default-language:    Haskell2010
-  ghc-options:         -threaded -rtsopts
-                       -Wall
-                       -fno-warn-orphans
-                       -O2
```

Comment:
Does `hpack` add `-O2` automatically?

Reply:
It doesn't, but I doubt it was ever actually needed. Will return it though.